setup.cfg

```diff
@@ -0,0 +1,2 @@
+[tool:pytest]
+addopts =-x -v -rw --durations=10 --cov=spyder --cov-report=term-missing
```

Comment:
These options are useful (or even necessary) when running the tests in the CI server, but in my opinion they are less useful when a user or developer runs the tests themselves. I think it would be better to move these options to the CI scripts (if possible).